apps/withdraw-pool/src/hooks/useUserPools.ts

```diff
@@ -128,6 +128,13 @@ export function useUserPools(chainId?: SupportedChainId, user?: string) {
             },
           }));
         });
+    },
+    {
+      shouldRetryOnError: false,
+      refreshWhenHidden: false,
+      refreshWhenOffline: false,
+      revalidateOnFocus: false,
+      revalidateOnMount: false,
```

Comment:
same as above, but I noticed here you also have `revalidateOnMount` set to false. Can you help me understand that decision?

Reply:
I was facing some UI blinks and I was trying to reduce the loading states 😅 I found the issue on another place so I will revert this.